aries_vcx/src/protocols/proof_presentation/prover/state_machine.rs

```diff
@@ -540,6 +547,47 @@ pub mod unit_tests {
         json!({}).to_string()
     }
 
+    mod build_messages {
+        use crate::messages::a2a::MessageId;
+        use crate::messages::proof_presentation::presentation_request::PresentationRequestData;
+        use crate::protocols::proof_presentation::prover::state_machine::{
+            build_presentation_message, build_problem_report,
+        };
+        use crate::utils::devsetup::{was_in_past, SetupMocks};
+
+        #[test]
+        #[cfg(feature = "general_test")]
+        fn test_prover_build_presentation_message() {
+            let _setup = SetupMocks::init();
+
+            let msg = build_presentation_message("12345", "{}".into()).unwrap();
+
+            assert_eq!(msg.id, MessageId("testid".into()));
+            assert_eq!(msg.thread.thid, Some("12345".into()));
+            assert!(was_in_past(
+                &msg.timing.unwrap().out_time.unwrap(),
+                chrono::Duration::milliseconds(100)
+            )
+            .unwrap());
+        }
+
+        #[tokio::test]
+        #[cfg(feature = "general_test")]
+        async fn test_prover_build_problem_report() {
+            let _setup = SetupMocks::init();
+            let msg = build_problem_report("12345", "foobar".into());
```

Comment:
Try to write the tests symmetrically, makes them more easier to read (missing newline)

Reply:
adjusted newlines across number of tests